packages/sui-studio/src/components/layout/_style.scss

```diff
@@ -40,7 +40,146 @@
       transform: translate3d(0, 0, 0);
     }
   }
-
+  .onehalf{
```

Comment:
maybe a whitespace before each `{` ? 😅

Reply:
👍 done, I've also removed .onehalf and .onethird, used only for my tests